website/docs/advanced/impression-data.md

```diff
@@ -3,7 +3,7 @@ title: Impression data
 ---
 
 :::info Availability
-The impression data feature was introduced in **Unleash 4.7**. Listening for events requires [an SDK that supports impression data events](../sdks/index.md#server-side-sdk-compatibility-table). Currently, it's only supported in the [Unleash Proxy client](../sdks/proxy-javascript.md) and [React Proxy client](../sdks/proxy-react.md).
+The impression data feature was introduced in **Unleash 4.7**. Listening for events requires [an SDK that supports impression data events](../sdks/index.md#server-side-sdk-compatibility-table).
```

Comment:
Good catch! Because we still don't have a compatibility table for frontend clients, I think we should still mention which frontend SDKs support it here. Something like this, perhaps?

```suggestion
The impression data feature was introduced in **Unleash 4.7**. Listening for events requires [an SDK that supports impression data events](../sdks/index.md#server-side-sdk-compatibility-table). In addition to server-side SDKs that support it, it's also supported in the JavaScript-based proxy clients.
```

Reply:
I reworded it a little bit, let me know if it's ok.